spec/javascripts/components/story/story_item_spec.js

```diff
@@ -80,4 +80,34 @@ describe('<StoryItem />', () => {
       expect(children).toHaveProp('title', title);
     });
   });
+
+  const render = props => {
+    const wrapper = shallow(<StoryItem {...props} />);
+    const expandedStory = wrapper.find('[data-id="expanded-story"]');
+    return { wrapper, expandedStory };
+  }
+
+  describe('when the story is accepted', () => {  
+    it('puts .Story--accepted on childrens prop className', () => {
+      const { expandedStory } = render({
+        story: storyFactory({
+          collapsed: false,
+          state: 'accepted'
+        })
+      });
```

Comment:
You could move this set up to `beforeEach` and let this `it` cleaner 😄 
```js
describe('when the story is accepted', () => {  
  let expandedStory

  beforeEach(() => {
    expandedStory = render({
      story: storyFactory({
        collapsed: false,
        state: 'accepted'
      })
    }).expandedStory;
  })

  it('puts .Story--accepted on childrens prop className', () => {
    expect(expandedStory).toHaveProp('className', 'Story--accepted');
  });
});
```

Reply:
To follow the Arrange-Act-Assert, it's better not to render stuff in an arranging (`beforeEach`) hook :smile:. It makes the code more explicit, making it a better documentation of how it's supposed to work.